stellarphot/utils/magnitude_transforms.py

```diff
@@ -644,7 +649,10 @@ def fake_it(one_image):
         color = cat["color"][cat_idx]
 
         # Impose some constraints on what is included in the fit
-        good_cat = ~(color.mask | cat_mag.mask) & (d2d.arcsecond < 1)
+        good_cat = ~(
+            getattr(color, "mask", False)
```

Comment:
Just curious why the switch to `getattr`, was this to allow default values?

Reply:
This was because the color, etc sometimes has a mask and some times doesn't, which is sort of a hassle.